app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -100,7 +100,7 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
           :data => {'toggle'        => 'modal',
                     'target'        => '#modal_d_div',
                     'function'      => 'miqCallAngular',
-                    'function-data' => '{"name": "showListener", "args": []}'})
+                    'function-data' => '{\"name\": \"showListener\", \"args\": []}'})
```

Comment:
The JSON messes up with string interpolation (not enough quotes to be used), so when there is JSON string it needs to be escaped.